README.md

```diff
@@ -599,17 +599,16 @@ Test server configuration is taken from 'test/config.yml'. If that file does not
 
     % cat test/config.yml
 
-    :csv:
-      :enable_module: true
-
     :foreman:
       :enable_module: true
       :host:          'http://katello:3000'
       :username:      'admin'
       :password:      'changeme'
-
     :katello:
       :enable_module: true
+    :csv:
+      :enable_module: true
+
```

Comment:
reordered the doc example config to match more typical order (foreman, then katello, and csv)